pupil_src/shared_modules/circle_detector.py

```diff
@@ -464,6 +465,46 @@ def get_nested_clusters(contours, hierarchy, min_nested_count):
             clusters[cluster[1]] = cluster
     return clusters.values()
 
+def getEllipsePts(e, num_pts=10):
```

Comment:
can we not use this here? https://docs.opencv.org/3.0-beta/modules/imgproc/doc/drawing_functions.html#ellipse2poly

Reply:
We now know this fn sucks.